docs/config/templates/ngdoc/api/module.template.html

```diff
@@ -69,7 +61,6 @@ <h2 id="known-issues">Known Issues</h2>
 {% endif %}
 
 
-{% if doc.componentGroups.length %}
```

Comment:
this is what is in dgeni-packages - so it looks like a breakage - but then it is re-fixed in a subsequent commit.